### DISCUSSION TITLE
Output gtf files

### DISCUSSION THREAD

**Discussion by sparthib** [ConesaLab/SQANTI3#306] [Q&A] [answered]
Hi there, 

I am using `ONT` reads, however, my `corrected.cds.gff` file has recorded the `source` as PacBio. 
`GL000194.1      PacBio  exon    53590   55676   .       -       .       transcript_id "ENST00000613230.1"; gene_id "ENSG00000277400.1";`

However `corrected.gtf` has recorded the original `source`. I have two questions. 
1. What's the difference between `corrected.gtf` and `corrected.gtf.cds.gff`?
2. Does having the wrong source (`pacbio` here) have any computational implications? Is there an argument where I can specify I am using ONT reads instead?

Thanks,
Sowmya

**Comment by pabloati** [comment, marked as answer]
Hi, sorry for the late reply. Regarding your questions:

1. The `corrected.gtf` file contains the whole annotation, while the `corrected.gtf.cds.gff` (now changed to `corrected.cds.gff3` only has the transcripts, exons and CDS information. 
2. Having the source changed to PacBio is a default action of SQANTI  (which will be solved in the future). It does not affect at all any of the computations nor the results.

Thanks
